routes/user.js

```diff
@@ -0,0 +1,186 @@
+module.exports = function(app) {
+  const config = require('../config')
+  const { url, resolve } = require("url")
+  const fetch = require("node-fetch")
+  const proxy = require("express-http-proxy")
+  const { check, validationResult } = require("express-validator")
+  const { sanitizeBody } = require("express-validator/filter")
+
+  //  don't enable routes unless user accounts enabled in config
+  if (config.get("USER_ACCOUNTS_ENABLED")) {
+    const doAPILogin = async body => {
```

Comment:
I think this should go to `model` layer and be part of `/lib/user.js`, e.g., see how `dms` routes uses model `/lib/dms.js`.